.changeset/beige-lemons-sin.md

```diff
@@ -0,0 +1,6 @@
+---
+'@bigcommerce/big-design': patch
+'@bigcommerce/docs': patch
```

Comment:
This is technically a visual breaking change, as it would change what a 3rd party app or view in the control panel would display. Can we chat outside of GitHub about the reason for this change?

As an aside, I'm thinking you can get away with a negative margin container to combat this margin if need be. Just thinking about a way to have a fix for this without adding a BC.